always320/downloader.py

```diff
@@ -16,24 +16,28 @@ def wget_get(url,use_url_lib=False,file_name=""):
     if use_url_lib:
         urllib_download(url,file_name)
     else:
-        downloaded = wget.download(url,file_name)
-        print("\n Downloaded file: " + downloaded)
-    return file_name
-
+        try:
+            import wget
```

Comment:
we should first try wget and if not found, fall back to urllib

Reply:
<h1>પોઈન્ટ છે, ભાણાં! </h1>

<small> You've got point!